bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/altair/AltairRule.java

```diff
@@ -15,6 +15,7 @@
 import edu.gemini.shared.util.immutable.Option;
 import edu.gemini.shared.util.immutable.Trio;
 import edu.gemini.shared.util.immutable.Tuple3;
+import edu.gemini.skycalc.Coordinates;
```

Comment:
@swalker2m Shouldn't we try to use the new target model for new code, I guess in this case it is ok given the size of the patch but we are just adding more places to port code later on